packages/dd-trace/src/exporters/agent/index.js

```diff
@@ -15,6 +15,12 @@ class AgentExporter {
     this._scheduler && this._scheduler.start()
   }
 
+  setUrl (url) {
+    url = new URL(url)
```

Comment:
This will throw if the URL is invalid.

Reply:
i wrapped it in a try/catch, logging the error as a warning.